scripts/generate-tree-sitter-libs.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# generate tree-sitter lib
+cd tree-sitter || exit
+make
+rm -rf "../src/main/resources/libtree-sitter.$1"
+cp "libtree-sitter.$1" ../src/main/resources
+cd ..
```

Comment:
This can definitely be improved! I just hacked this together to get the build green. I'm pretty sure we can move this whole script to gradle, it's just my gradle-fu was lacking.